src/vs/workbench/contrib/terminal/common/scripts/psreadline/PSReadLine.psd1

```diff
@@ -0,0 +1,19 @@
+@{
+RootModule = 'PSReadLine.psm1'
+NestedModules = @("Microsoft.PowerShell.PSReadLine.dll")
+ModuleVersion = '2.4.3'
+GUID = '5714753b-2afd-4492-a5fd-01d9e2cff8b5'
+Author = 'Microsoft Corporation'
+CompanyName = 'Microsoft Corporation'
+Copyright = '(c) Microsoft Corporation. All rights reserved.'
+Description = 'Great command line editing in the PowerShell console host'
+PowerShellVersion = '5.1'
```

Comment:
I'm surprised this is 5.1 🤔

Reply:
Don't look to closely...I found net6.0 references still existing in the codebase...for building its unit tests on macOS/Linux...which I then found hadn't been done in _years_.